kolibri_explore_plugin/models.py

```diff
@@ -1,7 +1,13 @@
+import json
```

Comment:
Ideally this file could do with a copyright/licensing header (as a separate commit), but that’s not a blocker for merging this PR because it’s a pre-existing issue.

Reply:
Good call. I think I'll add a prep commit that adds SPDX license identifiers to all the python files.